### PR TITLE
[Workaround] Skip opencl lstm test for Issue #692

### PR DESCRIPTION
**Pull request by ce1adon**
Workaround for issue #692

### CODE REVIEW

File: test/lstm_common.hpp [hunk 1]
```diff
@@ -2928,6 +2928,9 @@ struct lstm_basic_driver : test_driver
     {
 
 #if(MIOPEN_BACKEND_OPENCL == 1)
+        std::cout << "Skip test for Issue #692: " << std::endl;
```
Comment (atamazov) [inline test/lstm_common.hpp:2931]: I think we shall 
- formalize workaround
- use exit(...)
- avoid "unreachable code" warnings

**Review by atamazov — CHANGES_REQUESTED** [review]

**Comment by atamazov** [comment]
@ce1adon Please let me know if you do not have time for resolving https://github.com/ROCmSoftwarePlatform/MIOpen/pull/715#pullrequestreview-579784264, and I will do that.

Also I do not think it is worth to re-start CI testing until the review comments have been resolved. ;)

File: test/lstm_common.hpp (hunk 1 above)
Comment (ce1adon) [reply test/lstm_common.hpp:2931]: 1. What is current formal workaround format?
2.  will do
3.  where does the "unreachable code" warnings happen?

File: test/lstm_common.hpp [hunk 2]
```diff
@@ -2928,6 +2928,9 @@ struct lstm_basic_driver : test_driver
...
+        return;
+
         if(type == miopenHalf)
             exit(EXIT_SUCCESS);
```
Comment (atamazov) [inline test/lstm_common.hpp:2935]: This is unreachable code (perhaps CI will not complain)

File: test/lstm_common.hpp (hunk 1 above)
Comment (atamazov) [reply test/lstm_common.hpp:2931]: Thanks!

File: test/lstm_common.hpp [hunk 3]
```diff
@@ -2928,6 +2930,10 @@ struct lstm_basic_driver : test_driver
     {
 
 #if(MIOPEN_BACKEND_OPENCL == 1)
+#if WORKAROUND_ISSUE_692 == 1
```
Comment (atamazov) [inline test/lstm_common.hpp:2933]: [Note] FYI `#if WORKAROUND_ISSUE_692` is just enough. No need to fix this in this PR.

**Review by atamazov — APPROVED** [review]

**Pull request merged**